developments_build/sql/_status.sql

```diff
@@ -64,6 +64,7 @@ STATUS_translate as (
         a.date_lastupdt,
         a.address,
         a.occ_prop,
+        a.co_earliest_effectivedate,
```

Comment:
Will eventually get changed to `date_complete` to sync with PR #113, but those changes are not yet part of dev so fine to merge and update later.